mllib/src/main/scala/org/apache/spark/ml/evaluation/MultilabelClassificationEvaluator.scala

```diff
@@ -63,18 +63,18 @@ class MultilabelClassificationEvaluator (override val uid: String)
 
   setDefault(metricName -> "f1Measure")
 
-  final val metricClass: DoubleParam = new DoubleParam(this, "metricClass",
+  final val metricLabel: DoubleParam = new DoubleParam(this, "metricLabel",
```

Comment:
Same comment about maybe validating the metric value here from a list

Reply:
Do you means `MimaExcludes`?
The building is ok, we may not need to add `metricClass` into to it?